spec/scryfall/cards_spec.rb

```diff
@@ -28,5 +28,21 @@
         end
       end
     end
+
+    context 'random' do
+      it 'returns a JSON response containing the data for a random card' do
+        VCR.use_cassette('successful random search') do
+          expect(Scryfall::Cards.random(query: nil)['object']).to eq('card')
+        end
+      end
+    end
+
+    context 'random' do
+      it 'returns a JSON response containing the data for a filtered random card search' do
+        VCR.use_cassette('successful filtered random search') do
+          expect(Scryfall::Cards.random(query: 'Jace')['object']).to eq('card')
```

Comment:
I think there might be value in making sure the card data returned actually contains something from what we queried, which might necessitate narrowing down the query.  I'm not sure of the syntax, but doing something that restricts us to random cards with "Jace" in the name and then making sure the returned card object has a name containing "Jace" would do it.  This [rspec documentation on matchers](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers) should help.